packages/app/public/homepage/data.json

```diff
@@ -46,17 +46,26 @@
       {
         "iconUrl": "/homepage/icons/openshift-learning/ST-spring-boot_2x.png",
         "label": "Deploy Spring app on OpenShift",
-        "url": "https://developers.redhat.com/node/264191"
+        "url": "https://developers.redhat.com/node/264191",
+        "description": "This hands on experience will introduce you to developing on OpenShift. Learn how to access OpenShift from the command line, deploy an existing Spring application, and scale up your application to handle increased web traffic.",
+        "hours": 8,
+        "paths": 20
       },
       {
         "iconUrl": "/homepage/icons/openshift-learning/hello-kafka.png",
         "label": "Kafka 101",
-        "url": "https://developers.redhat.com/node/272761"
+        "url": "https://developers.redhat.com/node/272761",
+        "description": " Learn about the fundamentals of Apache Kafka. This tutorial covers basic concepts of Kafka and its components.",
+        "hours": 2,
+        "paths": 7
       },
       {
         "iconUrl": "/homepage/icons/openshift-learning/2020_RHD_Coding_imageA@2x.png",
         "label": "Red Hat OpenShift Data Science",
-        "url": "https://developers.redhat.com/node/272361"
+        "url": "https://developers.redhat.com/node/272361",
+        "description": "In this learning path, we use a pre-existing Jupyter Notebook project to start exploring data science. Project Jupyter offers the interactive JupyterHub and JupyterLab tools, which we'll introduce in this path.",
+        "hours": 1,
+        "paths": 4
```

Comment:
We can remove the learning path category from the home page and have another json file dedicated to the Learning path.

Reply:
Resolved in the latest commit.